aiida_quantumespresso/calculations/pp.py

```diff
@@ -13,28 +13,30 @@
 def validate_parameters(value, ctx=None):  # pylint: disable=unused-argument
     """Validate 'parameters' dict."""
     parameters = value.get_dict()
+    missing_namelists = 0
```

Comment:
For step1 only you don't need `parameters['PLOT']['iflag']`, and for step2 only you don't need `parameters['INPUTPP']['plot_num']`, but at least one of those two must be present. I keep count of how many are missing with this variable.